picard/ui/scripteditor.py

```diff
@@ -448,44 +602,92 @@ def toggle_wordwrap(self):
         else:
             self.ui.file_naming_format.setLineWrapMode(QtWidgets.QTextEdit.NoWrap)
 
+    def output_error(self, title, fmt, filename, msg):
+        """Log error and display error message dialog.
+
+        Args:
+            title (str): Title to display on the error dialog box
+            fmt (str): Format for the error type being displayed
+            filename (str): Name of the file being imported or exported
+            msg (str): Error message to display
+        """
```

Comment:
Since `title` is always `self.FILE_ERROR_TITLE`, perhaps just write a small wrapper to shorten lines a bit:

```python
def output_file_error(self, fmt, filename, msg):
      self.output_error(self.FILE_ERROR_TITLE, fmt, filename, msg)
```